assets/js/done-task.js

```diff
@@ -0,0 +1,28 @@
+import { CLICK_EVENT, HIDE_CLASS } from "/assets/js/constants.js";
+import createButton from "/assets/js/factory/createButton.js";
+
+function doneTaskEvent(done, edit, inputElement, tools, allTasks, id) {
+  done.addEventListener(CLICK_EVENT, () => {
```

Comment:
variable name incorrectly represents an action

```suggestion
function doneTaskEvent(doneButton, editButton, inputElement, tools, allTasks, id) {
  doneButton.addEventListener(CLICK_EVENT, () => {
  ...
```